src/stories/Forms/Fieldset/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Wrapper = styled.fieldset`
+  display: block;
+  min-width: 0;
+  max-width: 400px;
+  margin: 0 0 var(--spacing-32) 0;
+  padding: 0.01em 0 0 0;
+  border: 0;
+
+  legend {
+    display: table;
+    margin-bottom: var(--spacing-16);
+    padding: 0;
+    font-weight: bold;
+    font-size: 20px;
```

Comment:
Are we missing a font size variable we need? I guess we need to update the font sizes in generalt